CONTRIBUTING.md

```diff
@@ -2,23 +2,38 @@
 
 ## Steps to submit a PR
 
-Clone/Fork the repo
+- Clone/Fork the repo
+- Install dependencies: `npm install`
+- Ensure tests pass: `npm test`
+- Make changes and then make sure the tests still pass
+- Push your branch/fork and [submit a PR][pr]
+- Assign a [sparkboxer][contributors] to review your PR
 
-Install dependencies
+## Commit Style
+We use [conventional commits] to automate versioning of this package. Commit messages must be prefixed with a valid commit type.
 
-`npm install`
+Example:
+```sh
+feat: add new linting rule for ...
+```
 
-Ensure tests pass
+See the [conventional commits] page for further detail on available commit types and how to handle breaking changes.
 
-`npm test`
+## Creating a Release
+When making a change that requires a new version to be published, you'll want to run the release task before merging to master.
 
-Make changes and then make sure the tests still pass
+```sh
+$ npm run release
+```
 
-`npm test`
+This will:
+- Use all commit messages since the latest version tag to determine the correct new version
+- Increment the version in package.json and package-lock.json
+- Create a tagged commit with the new version
+- Push your branch (and new tag) to the remote
```

Comment:
I was looking though the conventional commits spec and `standard-version` and this all looks really nice! I was wondering, though, if it'd be worth adding any documentation for how to run this command if we wanted to override the version number (along the lines of `npm run release -- --release-as {XXX}`). Or do you think that goes against trying to standardize releases too much?

Reply:
No, I think that's a valid point, but I would consider it more of an escape hatch since the commits will cover the usual progression through versions. 🤔 I wouldn't want to encourage using it or add more mental overhead to the process just for edge cases. Would more of a general comment saying "it can do more" with a link to the standard-version docs be appropriate?

Reply:
Got it - that makes sense. I know the point of this is to standardize the process, so I can definitely be on board with not explicitly documenting how to override the automation. In that case, I'm on board with your idea and just adding a link to `standard-version`'s other commands along the lines of "it can do more" or "see `standard-version` for other commands."

Reply:
Pushed a commit for this.